ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.5.0f3
+m_EditorVersion: 5.5.1f1
```

Comment:
```
KeyNotFoundException: The given key was not present in the dictionary.
System.Collections.Generic.Dictionary`2[UnityEngine.Transform,LeapGuiSphericalSpace+Transformer].get_Item (UnityEngine.Transform key) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Collections.Generic/Dictionary.cs:150)
LeapGuiSphericalSpace.GetTransformer (UnityEngine.Transform anchor) (at Assets/LeapMotionModules/ElementRenderer/Scripts/Spaces/LeapGuiSphericalSpace.cs:39)
LeapGuiBakedRenderer.OnUpdateRenderer () (at Assets/LeapMotionModules/ElementRenderer/Scripts/Renderers/LeapGuiBakedRenderer.cs:77)
LeapGui.doLateUpdateRuntime () (at Assets/LeapMotionModules/ElementRenderer/Scripts/LeapGui.cs:333)
LeapGui.LateUpdate () (at Assets/LeapMotionModules/ElementRenderer/Scripts/LeapGui.cs:281)
```

To reproduce, just open the FeatureTest scene, hit play, and disable the Anchor of Constant Size component on the Anchor object.